.gitignore

```diff
@@ -51,9 +51,15 @@ docroot/modules/stanford
 docroot/themes/stanford
 docroot/profiles/stanford
 
-# Ignore custom theme build artifacts
+# Ignore custom theme build artifacts amd compiled files
 node_modules
 styleguide
+docroot/themes/humsci/humsci_airy/css/humsci_airy.css
```

Comment:
Shouldn't we also ignore compiled js here too?

Reply:
Fixed. Added the compiled js directory to `.gitignore` and removed the existing compiled code from the repo